python-test/features/steps/control_plane_policies.py

```diff
@@ -35,30 +36,38 @@ def clean_policies(context):
     delete_policies(token, policies_filtered_list)
 
 
+@given("that a policy already exists")
+def new_policy(context):
+    create_new_policy(context)
+    check_policies(context)
+
+
 def create_policy(token, policy_name, handler_label, handler, description=None, tap="default_pcap",
                   input_type="pcap", host_specification=None, filter_expression=None, backend_type="pktvisor"):
     """
 
     Creates a new policy in Orb control plane
 
-    :param filter_expression:
-    :param (str) host_specification: Subnets (comma separated) which should be considered belonging to this host,
-    in CIDR form. Used for ingress/egress determination, defaults to host attached to the network interface.
-    :param input_type: this must reference a tap name, or application of the policy will fail
-    :param tap: named, host specific connection specifications for the raw input streams accessed by pktvisor
+
     :param (str) token: used for API authentication
     :param (str) policy_name:  of the policy to be created
     :param (str) handler_label:  of the handler
-    :param (str) handler:  to be added
+    :param (str) handler: to be added
     :param (str) description: description of policy
+    :param tap: named, host specific connection specifications for the raw input streams accessed by pktvisor
+    :param input_type: this must reference a tap name, or application of the policy will fail
+    :param (str) host_specification: Subnets (comma separated) which should be considered belonging to this host,
+    in CIDR form. Used for ingress/egress determination, defaults to host attached to the network interface.
+    :param filter_expression: these decide exactly which data to summarize and expose for collection
     :param backend_type: Agent backend this policy is for. Cannot change once created. Default: pktvisor
     :return: (dict) a dictionary containing the created policy data
     """
     response = requests.post(base_orb_url + '/api/v1/policies/agent',
                              json={"name": policy_name, "description": description, "backend": backend_type,
-                                   "policy": {"kind": "collection", "input": {"tap": tap, "input_type": input_type}},
-                                   "config": {"host_spec": host_specification}, "filter": {"bpf": filter_expression},
-                                   "handlers": {"modules": {handler_label: {"type": handler}}}},
+                                   "policy": {"kind": "collection", "input": {"tap": tap, "input_type": input_type},
+                                              "handlers": {"modules": {handler_label: {"type": handler}}}},
+                                   "config": {"host_spec": host_specification}, "filter": {"bpf": filter_expression}},
+
```

Comment:
Again, let's create a variable for the json parameter instead of declaring it inline with the function call.
(I know this is just indentation of pre-existing code, but this is an easy improvement to be done)